src/app/admin/components/schedules/schedules.html

```diff
@@ -43,7 +43,10 @@ <h3>Types of Jobs</h3>
           <td>{{ job.name }}</td>
           <td>{{ job.description }}</td>
           <td>{{ job.jobDataMap.acbSpecific ? 'Yes' : 'No' }}</td>
-          <td><button class="btn btn-ai-success" ng-click="vm.editJob(job)" ng-if="job.jobDataMap.editableJobFields"><i class="fa fa-pencil-square-o"></i><span class="sr-only">Edit job for {{ job.name }}</span></button></td>
+          <td>
+            <button class="btn btn-ai-success" ng-click="vm.editJob(job)" ng-if="job.jobDataMap.editableJobFields"><i class="fa fa-pencil-square-o"></i><span class="sr-only">Edit job for {{ job.name }}</span></button>
+            <button class="btn btn-ai-success" ng-click="vm.createSimpleTrigger(job)" ng-if="job.group == 'systemJobs'"><i class="fa fa-bolt"></i><span class="sr-only">Create trigger for {{ job.name }}</span></button>
```

Comment:
```suggestion
            <button class="btn btn-ai-success" ng-click="vm.createSimpleTrigger(job)" ng-if="job.group === 'systemJobs'"><i class="fa fa-bolt"></i><span class="sr-only">Create trigger for {{ job.name }}</span></button>
```

Don't remember why javascript prefers `===` to `==`, but I know there's some reason for it